warehouse/models/mart/gtfs_quality/_mart_gtfs_quality.yml

```diff
@@ -30,9 +30,11 @@ models:
         name: check
         tests:
           - not_null
-          - relationships:
-              to: ref('stg_gtfs_quality__intended_checks')
-              field: check
+          # This is a useful test but dbt-metabase fails since
```

Comment:
I kind of think the answer here is going to be to instantiate the intended checks table in my refactor of the guidelines indexing stuff

Reply:
I think in this case it's also because the intended checks model would end up in `staging` which isn't in Metabase. I wasn't able to find a flag to tell dbt-metabase to ignore specific columns, only a whole model, which might work.